playground/tsconfig-json-load-error/__tests__/tsconfig-json-load-error.spec.ts

```diff
@@ -11,7 +11,7 @@ import {
 } from '~utils'
 
 const unexpectedTokenSyntaxErrorRE =
-  /^parsing .* failed: SyntaxError: Unexpected token.*\}.*/
+  /(\[vite:esbuild\] )*parsing .* failed: SyntaxError: Unexpected token.*\}.*/
```

Comment:
Is there a reason this is wrapped with a `( )*`? It seems to appear consistently for me locally.

Reply:
It doesn't seem to show up when running `test-serve`.
https://github.com/vitejs/vite/actions/runs/7592308559/job/20681516380

Reply:
Ah alright. This is fine by me then 👍